ChatBoatSwiftAI/ChatBoatSwiftAI/ViewController.swift

```diff
@@ -13,6 +13,7 @@ class ViewController: UIViewController {
     @IBOutlet weak var tv: UITextView!
 
     var chatBoatViewModel = ChatBoatViewModel()
+    var flag = 0
```

Comment:
Move this Properties in the ViewModel Class.
and make this Property of Type Boolean not Int.